### PR TITLE
OCM: added search operator version 2

### PR DESCRIPTION
**Pull request by xrajesh**
Signed-off-by: xavier dharmaiyan <xavier@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/search-v2-operator/stolostron-search-v2-operator-release-2.5.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      SOURCE_IMAGE_REF: search-v2-operator
+    env:
+      IMAGE_REPO: search-indexer
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/solostron"
+        export SELF="make -f Makefile.prow"
+        export OSCI_PUBLISH_DELAY="0"
```
Comment (schmidtd) [inline ci-operator/config/stolostron/search-v2-operator/stolostron-search-v2-operator-release-2.5.yaml:60]: You're going to want to define `OSCI_COMPONENT_VERSION` here

**Comment by xrajesh** [comment]
Will create a new PR

**Pull request closed without merging**